docs/application/dotnet/get-started/first-app.md

```diff
@@ -0,0 +1,491 @@
+<style>
+    .tabcontent img {
+        border: 1px solid #555;
+        max-width: 100% !important;
+        max-height: 100%;
+    }
+</style>
+
+# Create Your First Tizen .NET Application
```

Comment:
Hello Sagar. Thank you very much for the review. Now we have one first-app.md file instead of few ones. The first-app guide has own unique structure (tabbed layout).